openqabot/types/incident.py

```diff
@@ -144,3 +148,60 @@ def contains_package(self, requires: List[str]) -> bool:
                 if package.startswith(req) and package != "kernel-livepatch-tools":
                     return True
         return False
+
+    def has_failures(self, token) -> bool:
+        results = requests.get(
+            QEM_DASHBOARD + "/api/jobs/incident/" + str(self.id), headers=token
+        ).json()
+
+        failed_jobs = self.filter_failures(results)
+        if failed_jobs:
+            log.info("Found %s failed jobs for incident %s:", len(failed_jobs), self.id)
+            if log.isEnabledFor(LOG_DEBUG_LEVEL):
+                list(
+                    map(lambda job: self.log_debug_incident(job["job_id"]), failed_jobs)
```

Comment:
this should be standard `for` loop

Reply:
Yes, if the result is not used then it makes no sense to use map.

Reply:
> Yes, if the result is not used then it makes no sense to use map.

in reality `map` `reduce` and `filter` and `lambda` should be avoided in standard python

Reply:
"If it can be a `for` loop, it probably should be", seems to be true for most programming languages. 😁